templates/skeleton/app/routes/pages/$pageHandle.tsx

```diff
@@ -32,19 +32,54 @@ const seo: SeoHandleFunction<typeof loader> = ({data}) => ({
   description: data?.page?.seo?.description,
 });
 
+// wait what's going on here??
 export const handle = {
```

Comment:
This handle-based `seo` pattern is no longer recommended, we now return `seo` data from within the loader of each route. (See demo-store equivalent route)

As for the `handle` export, there is bit more info [here](https://remix.run/docs/en/1.14.3/route/handle)